catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/PaimonCatalogOperations.java

```diff
@@ -206,5 +284,17 @@ public boolean purgeTable(NameIdentifier identifier) throws UnsupportedOperation
   }
 
   @Override
-  public void close() throws IOException {}
+  public void close() throws IOException {
+    if (paimonTableOps != null) {
+      try {
+        paimonTableOps.close();
+      } catch (Exception e) {
+        throw new IOException(e.getMessage());
+      }
+    }
+  }
+
+  private static String currentUser() {
```

Comment:
seems should use  `UserGroupInformation.getCurrentUser().getUserName()` to get current user, @qqqttt123 @yuqi1129 help help to check it

Reply:
We should use `PrincipalUtils.getCurrentUserName` to get the current user name.

Reply:
got it.